services/networking/vpc/controls.yaml

```diff
@@ -26,24 +26,19 @@ controls:
     test_requirements:
       - id: CCC.VPC.C01.TR01
         text: |
-          Verify that default networks are not automatically created upon project initialization.
-        tlp_levels:
-          - tlp_red
-      - id: CCC.VPC.C01.TR02
-        text: |
-          Confirm that only custom networks with appropriate security controls are in place.
+          When a subscription is created, the subscription must not contain any
+          default network resources.
         tlp_levels:
           - tlp_red
 
   - id: CCC.VPC.C02
-    title: Limit External IP Addresses for Virtual Machines
+    title: Limit Resource Creation in Public Subnet
     objective: |
-      Restrict the assignment of external (public) IP addresses to virtual machines to reduce
-      exposure to the public internet and minimize attack surfaces.
+      Restrict the creation of resources in the public subnet with
+      direct access the internet to minimize attack surfaces.
     control_family: Network Security
     threats:
-      - CCC.VPC.TH02
-      - CCC.TH02 # Data is intercepted in transit (common threat)
+      - CCC.VPC.TH02  # Exposure of Resources to Public Internet
```

Comment:
CCM SEF-05 pertains to incident response metrics. IVS-09 (Secure Isolation) would be a better fit. It focuses on maintaining secure isolation of cloud tenants and their resources. Restricting resource creation in public subnets ensures that resources are isolated from public exposure, which aligns with IVS-09.